docs/spec.md

```diff
@@ -4027,26 +4306,36 @@ More formally:
   range provided by `exponent_bits`, the intermediate result overflows to
   infinity using the original sign or underflows to zero using the
   original sign.
+* For quantized types, the above three steps are performed using the original
+  floating-point value as `operand - zero_point(operand)) * scale(operand)` to
+  produce an intermediate result `float_result`. The final `result` is obtained
+  as:
+  * `rounded_result = round_nearest_even(float_result / scale(result))`
+  * `result = clamp(storage_min(result), rounded_result + zero_point(result),
+            storage_max(result)`
```

Comment:
Can this be simplified to `dequantize_op_quantize(reduce_precision, operand, type(result))`?

Reply:
done. Please take a look.